Rakefile

```diff
@@ -35,13 +34,11 @@ task :dist do
   # Use append_path if you want to build against a local clone
   #env.append_path 'asciidoctor/lib'
 
-  #env['asciidoctor'].write_to "build/asciidoctor.js#{compress ? '.gz' : nil}"
   asciidoctor = env['asciidoctor']
+  asciidoctor_src = env['asciidoctor/opal_ext'].source + env['asciidoctor'].source
   # NOTE hack to make version compliant with semver
-  asciidoctor_src = env['asciidoctor/core_ext/match_data'].source + env['asciidoctor'].source
   asciidoctor_src = asciidoctor_src.sub(/'VERSION', "(\d+\.\d+.\d+)\.(.*)"/, '\'VERSION\', "\1-\2"')
-  asciidoctor_src = asciidoctor_src.sub(/^( *)(self.\$require\("asciidoctor\/core_ext"\);)/,
-      %(\\1\\2\n\\1self.$require("asciidoctor/core_ext/match_data");))
+  asciidoctor_src = asciidoctor_src.sub(/^( *)(self\.\$require\("set"\);)/, %(\\1\\2\n\\1self.$require("asciidoctor/opal_ext");))
```

Comment:
I'm searching for this line of code but I'm not convinced that this is a good idea: https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L8

@mojavelinux Thoughts ?